### PR TITLE
create theme script for development

### PR DESCRIPTION
**Pull request by codeb-a**
### Description of work
- Creates a local script to support wiring tickets when component library work is not-yet-released
- Also exports a change to show images. Unrelated, but needed for a wiring ticket

### CODE REVIEW

**Comment by ModulesUnraveled** [comment]
@marcb4k @nJim @vinmassaro I just added you all to review this because I think we've all been doing similar work in various places. This script is to set up a local environment to be able to use code from the component library that's in a not-yet-released branch.

If you could take a look and provide feedback on how we can improve it based on the CI and other work you've done, that would be great!

File: scripts/local/dev-component-library-install.sh [hunk 1]
```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+GREEN='\033[1;32m'
+ENDCOLOR='\033[0m'
+
+read -p "Which branch? " BRANCH
+
+echo -e "${GREEN}Move into atomic and checkout develop"
+cd web/themes/contrib/atomic
+git checkout develop
```
Comment (marcb4k) [inline scripts/local/dev-component-library-install.sh:10]: I ran into an issue with npm not using the correct version. I did an ```nvm use``` from the atomic directory and that fixed it. So, perhaps after line 9, add an ```nvm use```

File: scripts/local/dev-component-library-install.sh [hunk 2]
```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+GREEN='\033[1;32m'
+ENDCOLOR='\033[0m'
+
+read -p "Which branch? " BRANCH
```
Comment (marcb4k) [inline scripts/local/dev-component-library-install.sh:6]: I'd add the repo name here, i.e. "Which branch from the component-library-twig repo" or something like that so it's clearer which repo you are asking for.

**Review by marcb4k — CHANGES_REQUESTED** [review]
Some small changes, but otherwise worked for me!

File: scripts/local/dev-component-library-install.sh (hunk 1 above)
Comment (ModulesUnraveled) [reply scripts/local/dev-component-library-install.sh:10]: hmm... that's assuming people have nvm installed, but I think that's a standard development tool - at least at 4K. I'll add it for now, and we can tweak it later, if we need to.

File: scripts/local/dev-component-library-install.sh (hunk 2 above)
Comment (ModulesUnraveled) [reply scripts/local/dev-component-library-install.sh]: Done

**Review by marcb4k — APPROVED** [review]
Chatted with Brian in Slack - Approved!

**Pull request merged**